sources/models/shared-defaults/aws-creds.toml

```diff
@@ -0,0 +1,20 @@
+[settings.aws]
+profile = "default"
+
+[services.aws]
+configuration-files = [
+  "aws-config",
+  "aws-credentials",
+]
+restart-commands = []
+
+[metadata.settings.aws]
+affected-services = ["aws"]
+
+[configuration-files.aws-config]
+path = "/root/.aws/config"
+template-path = "/usr/share/templates/aws-config"
+
+[configuration-files.aws-credentials]
+path = "/root/.aws/credentials"
+template-path = "/usr/share/templates/aws-credentials"
```

Comment:
Need an AddPrefixesMigration like this for the changes in this file:
```
fn run() -> Result<()> {
    migrate(AddPrefixesMigration(vec![
        "settings.aws.credentials",
        "settings.aws.config",
        "settings.aws.profile",
        "services.aws",
        "configuration-files.aws-config",
        "configuration-files.aws-credentials",
    ]))
}
```

Unfortunately, a rule of thumb is that almost any change reflected in these TOML model files needs to be covered by a migration.

(I think this can replace the `aws-creds` migration you have, but the `aws-creds-metadata` one is still needed to cover `metadata.settings.aws`.)